### PR TITLE
Adding new recipes for Encore, Stimulus & AssetMapper

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | TODO - WIP

Hi!

TODO:

* [x] Depends on https://github.com/symfony/flex/pull/975
* [x] `symfony/stimulus-bundle` needs to be properly released / under the `symfony` organization - probably tagged
* [x] Tag WebpackEncoreBundle 2.0
* [X] Should/can the recipes "conflict" with `symfony/flex` versions that don't support the new `add-lines` configurator?

Cheers!

### CODE REVIEW

**Comment by weaverryan** [comment]
This is ready to go! And needed to help the already-released StimulusBundle & WebpackEncoreBundle 2.0.

Failures are because CI tries to install on 5.4 and on 6.3... but using FrameworkBundle 6.2.

File: symfony/ux-translator/2.9/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "file": "assets/translator.js",
+            "content": "import { localeFallbacks } from '../var/translations/configuration';",
+            "position": "top",
+            "requires": "symfony/webpack-encore-bundle"
+        },
+        {
+            "file": "assets/translator.js",
+            "content": "import { localeFallbacks } from '@app/translations/configuration';",
+            "position": "top",
+            "requires": "symfony/stimulus-bundle"
```
Comment (kbond) [inline symfony/ux-translator/2.9/manifest.json:24]:
```suggestion
            "requires": "symfony/asset-mapper"
```

File: symfony/ux-translator/2.9/manifest.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            "file": "assets/translator.js",
+            "content": "export * from '../var/translations';",
+            "position": "bottom",
+            "requires": "symfony/webpack-encore-bundle"
+        },
+        {
+            "file": "assets/translator.js",
+            "content": "export * from '@app/translations';",
+            "position": "bottom",
+            "requires": "symfony/stimulus-bundle"
```
Comment (kbond) [inline symfony/ux-translator/2.9/manifest.json:36]:
```suggestion
            "requires": "symfony/asset-mapper"
```

**Review by kbond — CHANGES_REQUESTED** [review]
I _think_ these fixes are required.

**Review by kbond — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Pull request merged**